mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndexTest.java

```diff
@@ -18,6 +18,7 @@
  ****************************************************************/
```

Comment:
This one is quite different
- In the AbstractMessageSearchIndexTest, the message is added by manager layer -> then index.
- In the other ones(ElasticSearchListeningMessageSearchIndexTest.java  and LuceneMailboxMessageSearchIndexTest which does not extend for above abstract), the indexing is calling directly via index.add(...). That's why I added those test.